androidlibrary_lib/src/test/java/org/opendatakit/logging/desktop/ConfigurationTests.java

```diff
@@ -0,0 +1,35 @@
+package org.opendatakit.logging.desktop;
+
+import org.junit.Test;
+import org.opendatakit.logging.WebLoggerIf;
+import static org.junit.Assert.*;
+
+public class ConfigurationTests extends BaseLoggerTest {
```

Comment:
This class specifically tests the WebLoggerDesktopImpl. You should make that clearer in title.